nbviewer/static/less/slides.less

```diff
@@ -9,7 +9,8 @@ body {
 }
 
 .reveal {
-  font-size: 160%;
+  font-size: 130%;
```

Comment:
Must have misread it! Changed to match [nbconvert](https://github.com/jupyter/nbconvert/blob/master/nbconvert/templates/html/slides_reveal.tpl#L87)!